dialogue-serde/src/test/java/com/palantir/conjure/java/dialogue/serde/ConjureBodySerDeTest.java

```diff
@@ -196,6 +196,32 @@ public void testBinary_optional_empty() {
                 .isEmpty();
     }
 
+    @Test
+    public void testEmptyResponse_success() {
+        TestResponse response = new TestResponse();
+        response.code = 204;
+        BodySerDe serializers =
+                new ConjureBodySerDe(ImmutableList.of(WeightedEncoding.of(new StubEncoding("application/json"))));
+        serializers.emptyBodyDeserializer().deserialize(response);
+    }
+
+    @Test
+    public void testEmptyResponse_failure() {
+        TestResponse response = new TestResponse();
+        response.code = 400;
```

Comment:
if you merge in develop, I've just changed this to be a setter method, so `new TestResponse().code(400)`